substra/cli/printers.py

```diff
@@ -237,15 +242,20 @@ class ObjectivePrinter(AssetPrinter):
     )
     download_message = 'Download this objective\'s metric:'
 
-    def print_leaderboard_message(self, item):
+    def print_with_profile(self, profile):
+        if profile and profile != 'default':
+            return f'--profile {profile}'
+        return ''
```

Comment:
Since the `AlgoPrinter` class inherits from `AssetPrinter` it automatically gets the `print_with_profile` method, no need to override it.

Reply:
That's true, I remove it! 🙂